docs/finders/advanced.mdx

```diff
@@ -136,8 +136,9 @@ await $('Delete account').scrollTo().tap();
 
 And here's how `scrollTo()` works:
 
-1. Waits for at least 1 [Scrollable] widget to become visible
-2. Scrolls this `Scrollable` widget in its scrolling direction until the target
+1. Waits for at least 1 [Scrollable] widget (or whatever you provided in 
+   `view` argument) to become visible
+2. Scrolls this widget in its scrolling direction until the target
```

Comment:
Is this true?

Reply:
Yeah, why?

Reply:
> Scrolls this widget **in its scrolling direction** until the target

I thought that `scrollTo()` no longer infers the direction automatically. I recall we talked about it some time ago. But I might just not remember correctly.

Reply:
Ah, okay - that's awesome.